### DISCUSSION TITLE
Whats the window management software for Mac OS on the screenshot?

### DISCUSSION THREAD

**Discussion by 0xDmtri** [jrmoulton/tmux-sessionizer#9] [Q&A] [answered]
Could you please share the name of the window management software for Mac OS on the screenshot? Looks amazing!

**Comment by jrmoulton** [comment, marked as answer]
Thanks!

I'm using [yabai](https://github.com/koekeishiya/yabai)!

I'm using [Yabai Indicator](https://github.com/xiamaz/YabaiIndicator) as well
